### ISSUE TITLE
sourmash sketch fromfile missing in v4.3?

### ISSUE THREAD

**Issue by phiweger** [sourmash-bio/sourmash#2007]
Maybe I am using this wrongly:

```bash
# https://sourmash.readthedocs.io/en/latest/sourmash-sketch.html#bulk-sketch-construction-from-many-files
sourmash sketch fromfile input.csv -p dna -o db.zip
#  sketch: error: argument subcmd: invalid choice: 'fromfile' (choose from 'dna', 'rna', 'nucleotide', 'nt', 'protein', 'aa', 'prot', 'translate')
```

**Comment by mr-eyes** [comment]
Hi,

Would you please post the whole command line output, including the version number?

**Comment by phiweger** [comment]
Thx for the quick reply. The above literally is the command line output, and `sourmash --version` is `v4.3`, fresh conda install.

**Comment by mr-eyes** [comment]
@phiweger Thank you! I am confirming the bug in version 4.3.0.


```bash
mamba create -n issue_2007 bioconda::sourmash
conda activate issue_2007
sourmash --version
# sourmash 4.3.0

sourmash sketch fromfile input.csv -p dna -o db.zip
# sketch: error: argument subcmd: invalid choice: 'fromfile' (choose from 'dna', 'rna', 'nucleotide', 'nt', 'protein', 'aa', 'prot', 'translate')
```

---

However, it's fixed in the current unreleased version, `4.3.1`. In the meanwhile, you can install the latest version using the following command: `pip install git+https://github.com/sourmash-bio/sourmash`

You might need to install rust first `conda install conda-forge::rust`


Please let us know if you need further assistance.

**Comment by ctb** [comment]
hi, `sketch fromfile` is a new command and will be released in sourmash v4.4.0; see https://github.com/sourmash-bio/sourmash/issues/1968 for our draft notes. Sorry 'bout that, normally we put in version markers for new commands. and forgot for this one!

I'll update the docs in a bit; please leave this issue open. Thanks!

**Comment by ctb** [comment]
see #2009.

**Comment by mr-eyes** [comment]
Resolving some confusion in my previous comments and also for future reference.

sourmash docs with [`readthedocs/latest`](https://sourmash.readthedocs.io/en/latest/) is for the [`github/latest`](https://github.com/sourmash-bio/sourmash/tree/latest) branch which may or may not reflect the latest sourmash release.

`fromfile` was a not-yet-implemented feature in sourmash <`v4.4.0`, and the latest released version is `v4.3.0`
